contracts/CounterApp.sol

```diff
@@ -34,8 +31,12 @@ contract CounterApp is AragonApp {
      * @notice Decrement the counter by `step`
      * @param step Amount to decrement by
      */
-    function decrement(uint256 step) auth(DECREMENT_ROLE) external {
+    function decrement(uint256 step) external auth(DECREMENT_ROLE) {
         value = value.sub(step);
         emit Decrement(msg.sender, step);
     }
+
+    function initialize() public onlyInit {
+        initialized();
+    }
```

Comment:
Ahh yeah, we typically keep this at the top because it's acting as the "constructor". Let's use a `solium-ignore` directive to not have this fail linting.

Reply:
Added exception in ethlint :+1: